gtsam/3rdparty/cephes/dllexport.h

```diff
@@ -0,0 +1,60 @@
+/* ----------------------------------------------------------------------------
```

Comment:
Copy-paste? if this is just an Windows issue just add the required export for Windows

Reply:
Copy paste. I didn't want to change something that already working.

Reply:
@talregev I'm making a Cephes related PR, I will update it to include the proper `extern` definitions.

Reply:
@varunagrawal Thank you! I will rebase your PR after it will be on dev.
Can you review the rest of the PR?